test/micro/org/openjdk/bench/java/security/SSLHandshake.java

```diff
@@ -51,6 +51,9 @@
 @BenchmarkMode(Mode.Throughput)
 @OutputTimeUnit(TimeUnit.SECONDS)
 @State(Scope.Benchmark)
+@Warmup(iterations = 5, time = 1)
+@Measurement(iterations = 5, time = 1)
+@Fork(value = 3)
```

Comment:
should the `@Warmup`, `@Measurement`, and `@Fork` be removed from the `doHandshake()` benchmark on lines 115-117 below so that these take effect?

https://github.com/openjdk/jdk/blob/c2cbeb3ee875936c98bb15ec32d692f7d866df76/test/micro/org/openjdk/bench/java/security/SSLHandshake.java#L111-L115

Reply:
Good observation. This means this benchmark was already tuned, and the pre-existing tuning overrides the setting I thought I was testing with. I've effectively reverted the changes but moved the annotations to class scope for consistency.